app/controllers/v0/sign_in_controller.rb

```diff
@@ -347,6 +345,16 @@ def create_login_code(state_payload, user_info, credential_level) # rubocop:disa
              content_type: 'text/html'
     end
 
+    def create_user_code_map(user_attributes, state_payload, verified_icn, request_ip)
```

Comment:
Probably a nit, but to keep the controllers clean, should we move this method to a helper or module?

Reply:
I would say yes but this is going to be completely removed in an upcoming PR 😄